### ISSUE TITLE
Multiple issues

### ISSUE THREAD

**Issue by da1cru1** [Steppschuh/RemoteControlServer#96]
Hello,
I contacted you a bit ago when I paid for and downloaded the full-version of the Remote App. I wasn't am am still upset you can't turn off the Left-Click Tap somehow through the settings, (for an app that costs $5 and seems to be a decent, (and being dedicated to One thing), I would Never have thought this could be an Afterthought, nor should it be hard to write the setting in for a new update which allows one to do so, [As Not Everyone uses the Tap Feature). Your response was "Most people prefer, or make use of Tapping). But it is what it is........... 
Also from the beginning, It won't allow me to Run the Remote from Start-Up, (it says you must be an Admin, yet I'm the only Acc't on this laptop as Noone else uses it; So I don't understand that either), which is a bit bothersome considering you actually included this in the Settings, yet doesn't work, but can't (or won't) include a way to turn off the Tap controls.
Now my latest troubles which Just started happening 3/16 about 8:00pm EST, When I (manually) turn the "server" on, on my laptop, Before I can even connect my phone, 2 things happen almost simultaneously. First after "loading", and the little Icon appears saying the Server is on waiting to connect to the App, The Phone looking thing pops up Onscreen and a message comes up saying the Application has stopped working-While a message pops up from the Icon saying there is an Update to the Remote Control App.
So simply put, it opens, and at the same time says there's an Update Available, and the App Stops working and Closes at the same time, therefore Not allowing me to "Update the Server/App"!!!!
What do I do to remedy this situation?
Thanks,

**Comment by Steppschuh** [comment]
Hey,
As already stated, the tap-to-click functionality won't change. Independent of the amount of money you've spent, the app will stay the same and only unlock the premium remotes.

For the autostart setting to take effect, the server needs to be started with administrator rights (as said). That's needed, because executables without these rights are not allowed to write certain registry entries. The fact that you are signed in as admin doesn't mean that you execute everything with admin rights. Refer to http://www.sevenforums.com/tutorials/11841-run-administrator.html
If you don't have admin rights, simply drag a shortcut to the server into the autostart directory.

To get the latest server version, run the "update" executable (with admin rights), or uninstall the server and download the latest version from the website. You can find detailed instructions here:
https://github.com/Steppschuh/RemoteControlServer/issues/7
